data/rim jobs.txt

```diff
@@ -0,0 +1,144 @@
+# Copyright (c) 2017 by Michael Zahniser
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+mission "Southbound Shipment [0]"
+	name "Shipment to <planet>"
+	job
+	repeat
+	description "Deliver <cargo> to <destination>, the home of Southbound Shipyards, the largest ship manufacturer in the region. Payment is <payment>."
+	cargo "weapons" 5 2 .1
+	to offer
+		random < 50
+	source
+		near "Zug" 2 10
+		attributes "rim" "south"
+	destination "Zug"
+	npc
+		government "Pirate"
+		personality heroic harvests plunders
+		fleet "Small Southern Pirates"
+	on complete
+		payment 0 160
+		dialog "You drop off your cargo of <commodity> and collect your payment of <payment>."
+
+mission "Southbound Shipment [1]"
+	name "Shipment to <planet>"
+	job
+	repeat
+	description "Deliver <cargo> to <destination>, the home of Southbound Shipyards, the largest ship manufacturer in the region. Payment is <payment>."
+	cargo "ship alloys" 5 2 .1
+	to offer
+		random < 50
+	source
+		near "Zug" 2 10
+		attributes "rim" "south"
+	destination "Zug"
+	npc
+		government "Pirate"
+		personality heroic harvests plunders
+		fleet "Small Southern Pirates"
+	on complete
+		payment 0 160
+		dialog "You drop off your cargo of <commodity> and collect your payment of <payment>."
+
+
+mission "Kraz Shipment [0]"
+	name "Shipment to <planet>"
+	job
+	repeat
+	description "Deliver <cargo> to <destination>, the home of Kraz Cybernetics, the largest weapons manufacturer in the region. Payment is <payment>."
+	cargo "weapons parts" 5 2 .1
+	to offer
+		random < 50
+	source
+		near "Zug" 2 10
+		attributes "rim" "south"
+	destination "Zug"
+	npc
+		government "Pirate"
+		personality heroic harvests plunders
+		fleet "Small Southern Pirates"
+	on complete
+		payment 0 160
+		dialog "You drop off your cargo of <commodity> and collect your payment of <payment>."
+
+mission "Han Sizer Month [0]"
+	name `Han Sizer Celebration`
+	job
+	repeat
+	description `Han Sizer month commemorates the first explorer of the Rim. To celebrate, citizens visit as many planets in the Rim as they can in the month. Bring <fare> to all the marked systems and return to <destination> for <payment>.`
```

Comment:
I love this mission; so creative!

Reply:
It might be better to just throw the complete information on the nature of Han Sizer Month into some planet description and only mention in the job descriptions that people are moving around to celebrate it, in the same style as the Coalition Games jobs.

Also, you might want to decapitalize "celebration". And/or capitalize the first "month".

Reply:
I could possibly do that. Longjump, the first planet in the Rim, has an abysmally tiny spaceport and planet description that could use some filling.

Reply:
I've added information about Han Sizer month to the description of Longjump.